content/templates/content/artisanprogram.html

```diff
@@ -5,20 +5,23 @@
 {% if forloop.first %}
 <div class="row">
     {% if program.img %} <!--  if image 4, 8 col -->
-    <div class="col-md-4">
-        <h1 class="childrenstitle text-center text-uppercase">{{program.title}}</h1>
-    </div>
-    <div class="col-md-8">
-        <img class="center-block img-responsive programimg" src="{{program.img.url}}">
+    <div class="col-md-7 col-md-push-5">
+      <img class="center-block img-responsive ethicsimage" src="{{ program.img.url}}">
```

Comment:
can you add space to the end of the `}}` as well?

`{{ program.img.url }}` instead of `{{ program.img.url}}`